templates/zerver/api/real-time-events.md

```diff
@@ -50,9 +50,9 @@ client.call_on_each_event(lambda event: sys.stdout.write(str(event) + "\n"))
 
 {end_tabs}
 
-## Arguments
+## Parameters
 
-You may also pass in the following keyword arguments to `call_on_each_event`:
+You may also pass in the following keyword parameters to `call_on_each_event`:
```

Comment:
We are talking about a Python function here, so the term "arguments" is correct here.

Reply:
Done